Dockerfile

```diff
@@ -39,13 +39,19 @@ RUN set -x && \
     dnf -y builddep /opt/ci/dnf-behave-tests/requirements.spec && \
     pip3 install -r /opt/ci/dnf-behave-tests/requirements.txt
 
+RUN set -x && \
+    dnf -y install dbus-daemon python3-dbus polkit
```

Comment:
Not here anymore, please. Add the deps to `dnf-behave-tests/requirements.spec` (see the step above).